drivers/usb/udc/udc_stm32.c

```diff
@@ -1017,24 +1017,16 @@ static void priv_pcd_prepare(const struct device *dev)
 
 	memset(&priv->pcd, 0, sizeof(priv->pcd));
 
-	/* Default values */
 	priv->pcd.Init.dev_endpoints = cfg->num_endpoints;
 	priv->pcd.Init.ep0_mps = cfg->ep0_mps;
 	priv->pcd.Init.speed = PCD_SPEED_FULL;
-
+	priv->pcd.Init.phy_itface = DT_INST_STRING_UPPER_TOKEN(0, physel);
```

Comment:
Remove this line (from commit "drivers: usb: udc_stm32: refactor USB instance assignment").
`physel` porperty is handled in the next commit ("drivers: usb: udc_stm32: refactor USB phy_itface assignment").